src/index.d.ts

```diff
@@ -350,6 +350,12 @@ declare namespace RamdaAdjunct {
         mergeProps(ps: Array<string>): (obj: Object) => Object;
 
         /**
+         * Merge objects under corresponting paths.
+         */
+        mergePaths(ps: Array<Array<string>>, obj: Object): Object;
```

Comment:
Signature of paths is 
```typescript
paths(ps: Array<Array<string | number>>, obj: Object): Array<any>;
```
They can also process arrays indexes that are numbers. Let's mirror it into mergePaths also

Reply:
hm, really, objects can be in arrays, neat